lib/config/presets/internal/workarounds.ts

```diff
@@ -292,4 +293,14 @@ export const presets: Record<string, Preset> = {
       },
     ],
   },
+  ubuntuDockerVersioning: {
+    description: 'Use ubuntu versioning for `node` docker images.',
+    packageRules: [
+      {
+        matchDatasources: ['docker'],
+        matchPackageNames: ['ubuntu'],
```

Comment:
```suggestion
        matchDepNames: ['ubuntu'],
```
package name is probably the full name